### ISSUE TITLE
Error Adjust lighting

### ISSUE THREAD

**Issue by 0scarZulu** [MathiasHun/FS22_Real_Vehicle_Breakdowns_BETA#79]
Hello :) 

I find it inappropriate that if you have an error with the lighting, you can no longer use any light at all. This also means that you can no longer use the work lights. Perhaps it is possible to switch on a single light at random. Or if that is not possible, then the normal light and the working light can be switched on separately. I think that would be more realistic. 
Cheers

**Comment by MathiasHun** [comment]
Hello.
Since it is possible to switch the lights all the way through using a single button, that is why is currently not possible.

**Comment by 0scarZulu** [comment]
But I can switch the work lights separately with Numpad 5 and 6, for example. So I could use the 3 lights separately, couldn't I? So low beam, front work light and rear work light, right?

**Comment by MathiasHun** [comment]
Yes, but the front and rear work lights can also be switched on with the F button. So I disabled the Numpad 5 and 6 buttons in vain.
